### PR TITLE
Add variation of CAPG to test against master and stable of CAPI and CABPK

### PR DESCRIPTION
**Pull request by dims**
Change-Id: I50b51dda0c9e9c09ddf45b3e93725bc46d000878

### CODE REVIEW

**Comment by dims** [comment]
we'll need https://github.com/kubernetes-sigs/cluster-api-provider-gcp/pull/203 to merge first

File: config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-ci.yaml [hunk 1]
```diff
@@ -44,6 +44,11 @@ periodics:
   spec:
     containers:
       - image: gcr.io/k8s-testimages/kubekins-e2e:v20191001-7ca508a-master
+        env:
```
Comment (dims) [inline config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-ci.yaml]: @detiber does `master` work here? based on the change you suggested in https://github.com/kubernetes/kubernetes/issues/81760 ?

Comment (detiber) [reply config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-ci.yaml]: I believe it should :)

**Pull request merged**